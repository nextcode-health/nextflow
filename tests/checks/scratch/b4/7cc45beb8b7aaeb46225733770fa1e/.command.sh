#!/bin/bash -ue
echo indx: 3
echo proc: printVars
echo exec: local
echo cpus: 2
echo penv: mpi
echo time: 1d
echo memory: 1 GB
