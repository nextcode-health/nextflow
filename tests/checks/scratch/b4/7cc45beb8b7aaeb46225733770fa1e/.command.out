indx: 3
proc: printVars
exec: local
cpus: 2
penv: mpi
time: 1d
memory: 1 GB
